networkx/classes/tests/test_digraph_historical.py

```diff
@@ -59,11 +59,11 @@ def test_neighbors(self):
         assert_equal(sorted(G.neighbors('C')),['D'])
         assert_equal(sorted(G['C']),['D'])
         assert_equal(sorted(G.neighbors('A')),['B', 'C'])
-        assert_equal(sorted(G.neighbors_iter('A')),['B', 'C'])
-        assert_equal(sorted(G.neighbors_iter('C')),['D'])
         assert_equal(sorted(G.neighbors('A')),['B', 'C'])
+        assert_equal(sorted(G.neighbors('C')),['D'])
+        assert_equal(sorted(G.neighbors('A')),['B', 'C'])
+        assert_raises(nx.NetworkXError,G.neighbors,'j')
         assert_raises(nx.NetworkXError,G.neighbors,'j')
```

Comment:
This ends up with many identical tests repeated. We probably want to remove duplicates.